### PR TITLE
Oracle DB Audit Data Connector

### PR DESCRIPTION
**Pull request by vu-socprime**
## Proposed Changes
  - Add Oracle DB Audit Data Connector
  - Add Oracle DB Audit Workbook

### CODE REVIEW

**Comment by v-jayakal** [comment]
@vu-socprime , Please modify the "fromTemplateId" of workbook to refer Oracle DB instead of generic "sentinel-UserWorkbook", you can check other workbooks for reference.

File: DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json [hunk 1]
```diff
@@ -0,0 +1,124 @@
+{
+    "id": "OracleDatabaseAudit",
+    "title": "Oracle Database Audit",
+    "publisher": "Oracle",
+    "descriptionMarkdown": "The Oracle DB Audit data connector provides the capability to ingest [Oracle Database](https://www.oracle.com/database/technologies/) audit events into Azure Sentinel through the syslog. Refer to [documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/introduction-to-auditing.html#GUID-94381464-53A3-421B-8F13-BD171C867405) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **OracleDatabaseAuditEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-OracleDatabaseAudit-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Oracle Database Audit Events",
```
Comment (v-jayakal) [inline DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json:10]: Legend value should be OracleDatabaseAudit

**Comment by v-jayakal** [comment]
@vu-socprime, There is an additional parentheses in the text for Parser, Can you please remove?
Text:  This data connector depends on a parser based on Kusto Function **OracleDatabaseAuditEvent** to work as expected.  [Follow steps to get this Kusto Function]((https://aka.ms/sentinel-OracleDatabaseAudit-parser)"
See the parentheses before https: here.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

**Comment by vu-socprime** [comment]
> @vu-socprime , Please modify the "fromTemplateId" of workbook to refer Oracle DB instead of generic "sentinel-UserWorkbook", you can check other workbooks for reference.

done

**Comment by vu-socprime** [comment]
> @vu-socprime, There is an additional parentheses in the text for Parser, Can you please remove?
> Text: This data connector depends on a parser based on Kusto Function **OracleDatabaseAuditEvent** to work as expected. [Follow steps to get this Kusto Function]((https://aka.ms/sentinel-OracleDatabaseAudit-parser)"
> See the parentheses before https: here.

done

File: DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json [hunk 2]
```diff
@@ -0,0 +1,114 @@
+{
+    "id": "OracleDatabaseAudit",
+    "title": "Oracle Database Audit",
+    "publisher": "Oracle",
+    "descriptionMarkdown": "The Oracle DB Audit data connector provides the capability to ingest [Oracle Database](https://www.oracle.com/database/technologies/) audit events into Azure Sentinel through the syslog. Refer to [documentation](https://docs.oracle.com/en/database/oracle/oracle-database/21/dbseg/introduction-to-auditing.html#GUID-94381464-53A3-421B-8F13-BD171C867405) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **OracleDatabaseAuditEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-OracleDatabaseAudit-parser)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "Syslog(OracleDatabaseAudit)",
```
Comment (v-jayakal) [inline DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json:10]: Please remove Syslog, "legend": "OracleDatabaseAudit" is the value expected.

File: DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Sources",
+            "query": "OracleDatabaseAuditEvent\n | summarize count() by SrcDvcHostname\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "Syslog(OracleDatabaseAudit)",
```
Comment (v-jayakal) [inline DataConnectors/OracleDatabaseAudit/Connector_OracleDatabaseAudit.json:22]: Please keep the space next to Syslog, There should not be any space within "OracleDatabaseAudit.

**Review by v-jayakal — CHANGES_REQUESTED** [review]

**Review by v-jayakal — CHANGES_REQUESTED** [review]
@vu-socprime ,Please resolve merge confilcts of Workbook metadata file, Thanks.

**Comment by v-jayakal** [comment]
@vu-socprime ,Please resolve merge conflicts of Workbook metadata file, Thanks.

**Comment by vu-socprime** [comment]
> @vu-socprime ,Please resolve merge conflicts of Workbook metadata file, Thanks.

@v-jayakal , resolved

**Review by v-jayakal — APPROVED** [review]

**Pull request merged**